sat/apiclient/fas.py

```diff
@@ -319,13 +333,12 @@ def _create_temporary_snapshot(self, xnames=None):
                       'devices' field.
                     - The FAS did not indicate if the snapshot was ready.
         """
-        now, later = _now_and_later(10)
+        now = create_snapshot()
```

Comment:
just remove `create_snapshot` and put the code that gets the date here

```suggestion
        now = datetime.utcnow().replace(microsecond=0)
```